pkg/controller/node/nodecontroller.go

```diff
@@ -79,6 +79,16 @@ var (
 		Key:    metav1.TaintNodeNotReady,
 		Effect: v1.TaintEffectNoExecute,
 	}
+
+	// TODO(resouer) move this to metav1 well_known_labels.go after #42406 get merged.
+	// TaintNodeNoSchedule would be automatically added by node controller when
+	// node runs into a unschedulable condition and removed when node becomes schedulable.
+	TaintNodeNoSchedule = "node.beta.kubernetes.io/noSchedule"
```

Comment:
I asking this because I want to know for my PR [Move TaintBasedEvictions feature from alpha to beta (core part)](https://github.com/kubernetes/kubernetes/pull/45402), do I need to enable the `TaintBasedEvictions` by default? ;-)